src/components/tables/PoolsTable/TokenPills/HiddenTokensPills.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="relative flex mr-2 my-1">
+    <div class="pill" :style="{ zIndex: tokens.length }">
```

Comment:
The is the "top-level" pill - so the `zIndex` value is the highest. (other layers will have lower `zIndex`)